lib/foreman_tasks/cleaner.rb

```diff
@@ -149,19 +187,20 @@ def prepare_filter
       filter_parts.select(&:present?).join(' AND ')
     end
 
-    def start_tracking_progress
-      if verbose
-        @current = 0
-        @total = tasks.size
-        say "#{@current}/#{@total}", false
-      end
+    def start_tracking_progress(name, total = tasks.size)
+      say "About to remove #{total} #{name}(s)"
+      @current = 0
+      @total = total
+      say "#{@current}/#{@total}", false if verbose
     end
 
     def report_progress(chunk)
-      if verbose
-        @current += chunk.size
-        say "#{@current}/#{@total}", false
-      end
+      @current += chunk.count
+      say "#{@current}/#{@total}", false if verbose
+    end
+
+    def report_done(name)
+      say "Deleted #{@current} #{name}(s)"
```

Comment:
With orphaned tasks, I was getting `Deleted 0 orphaned execution plans` even though there has been reported (and deleted` 1 orhphaned execution plan to be deleted